### DISCUSSION TITLE
Creating Photorealistic Images from Sketches

### DISCUSSION THREAD

**Discussion by Laterport** [lllyasviel/Fooocus#2334] [Q&A]
Hello everyone! Could you please advise me on preparing a photorealistic image using a neural network from my sketch? What should the sketch look like for the neural network to draw the photo based on it as closely and detailed as possible? Also, what tool is used in Fooocus for this? Are there any recommendations for its settings? The drawings are planned to be of people.

**Comment by mashb1t** [comment]
Fooocus doesn't support lineart /sketch controlnet, but canny (basically the outline of an object). You can try to upload a sketch in either vary or to image prompt with advanced checkbox (in image brompt, bottom) checked and selected PyraCanny for the input image.
If this doesn't work you may try A1111 with ControlNet lineart or sketch. See https://github.com/lllyasviel/ControlNet-v1-1-nightly
